pkg/config/system_probe.go

```diff
@@ -53,6 +53,7 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.profile_dd_url"), "", "DD_SYSTEM_PROBE_INTERNAL_PROFILING_DD_URL", "DD_APM_INTERNAL_PROFILING_DD_URL")
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.api_key"), "", "DD_SYSTEM_PROBE_INTERNAL_PROFILING_API_KEY", "DD_API_KEY")
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.env"), "", "DD_SYSTEM_PROBE_INTERNAL_PROFILING_ENV", "DD_ENV")
+	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.period"), 5, "DD_SYSTEM_PROBE_INTERNAL_PROFILING_PERIOD")
```

Comment:
Having this be a number of minutes, means we cannot ever go lower than 1 minute. Viper does support duration formats, so we could use `GetDuration` instead and have the default be `"5m"`?

If 1 minute being the lower-bound is OK, then I might suggesting renaming this to `period_minutes` to be extra clear on the units.

Reply:
Done ✅